ext/opcache/ZendAccelerator.c

```diff
@@ -3171,14 +3171,8 @@ static zend_result accel_post_startup(void)
 		 && zend_jit_check_support() == SUCCESS) {
 			size_t page_size;
 
-# ifdef _WIN32
-			SYSTEM_INFO system_info;
-			GetSystemInfo(&system_info);
-			page_size = system_info.dwPageSize;
-# else
-			page_size = getpagesize();
```

Comment:
Linux manual recommends using `sysconf(_SC_PAGESIZE)` instead of `getpagesize()` for portability.
The code is referenced from https://github.com/jemalloc/jemalloc/blob/master/src/pages.c#L417
The comment shows the reason for it, it can avoid a useless syscall on FreeBSD.